libvast/src/system/accountant.cpp

```diff
@@ -41,31 +40,10 @@ namespace {
 using accountant_actor = accountant_type::stateful_base<accountant_state>;
 constexpr std::chrono::seconds overview_delay(3);
 
-void init(accountant_actor* self, const path& filename) {
+void init(accountant_actor* self) {
   auto& st = self->state;
-  if (!exists(filename.parent())) {
-    auto t = mkdir(filename.parent());
-    if (!t) {
-      VAST_ERROR(self, to_string(t.error()));
-      self->quit(t.error());
-      return;
-    }
-  }
-  VAST_DEBUG(self, "opens log file:", filename.trim(-4));
-  auto& file = self->state.file;
-  file.open(filename.str());
-  if (!file.is_open()) {
-    VAST_ERROR(self, "failed to open file:", filename);
-    auto e = make_error(ec::filesystem_error, "failed to open file:", filename);
-    self->quit(e);
-    return;
-  }
-  file << "timestamp\tnode\taid\tactor_name\tkey\tvalue\n";
-  if (!file)
-    self->quit(make_error(ec::filesystem_error));
-  VAST_DEBUG(self, "kicks off flush loop");
-  self->send(self, flush_atom::value);
 #if VAST_LOG_LEVEL >= CAF_LOG_LEVEL_INFO
```

Comment:
This is different from your changes, but I find this pre-processor conditional not very intuitive. To me, it reads like "only perform this when logging is enabled". Can we have a clearer macro for that?

Reply:
> To me, it reads like "only perform this when logging is enabled".

That is correct however. We don't want to print the heartbeat if verbosity is set to `WARNING` or lower.

Reply:
Okay, I think I understand the technical aspect. But wouldn't you agree that this is a hard-to-read of conveying "logging enabled?"